code/modules/reagents/chemistry/reagents/impure_reagents/impure_toxin_reagents.dm

```diff
@@ -0,0 +1,73 @@
+//Reagents produced by metabolising/reacting fermichems inoptimally these specifically are for toxins
+//Inverse = Splitting
+//Invert = Whole conversion
+//Failed = End reaction below purity_min
+
+////////////////////TOXINS///////////////////////////
+
+//Lipolicide
+//impure
```

Comment:
Can you just combine all these comments into `/// Impure version of Lipolicide` etc?

Reply:
All combined into single lines now